proxygen/lib/http/codec/compress/experimental/simulator/CompressionSimulator.cpp

```diff
@@ -197,62 +227,72 @@ void CompressionScheme::runLoopCallback() noexcept {
   simulator_->flushSchemePackets(this);
 }
 
+void CompressionSimulator::flushPacket(CompressionScheme* scheme) {
+  if (scheme->packetBlocks.empty()) return;
```

Comment:
nit - we require braces for all control blocks.  Again, only change if you are making another rev.